src/server/routes/summary/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const { entitySearch } = require('./entity');
+
+router.get('/entity/search', function (req, res) {
+  let tokens = req.query.q.trim().split(' ');
+  entitySearch( tokens ).then( r => res.json( r ) );
```

Comment:
Not sure what the behaviour is when you res.json( undefined )

Reply:
There should be a `.catch()` that then sends an http error code via `res`.

Reply:
@jvwong add error handling and then we can just merge this.

Reply:
If I was the client, I think I would just care about whether the request succeeded or not.  On the server, it might be useful to log the error so that we can know the actual cause of failure.

Reply:
It's usually best to not just send the same error object to the client.  In some cases, it could contain private information.  For this case, privacy probably doesn't matter.

The other general consideration is context, like Dylan mentioned.  It's good to summarise the error in plain readable language that accommodates the client's usecase.  

E.g. `res.status(500).send('Failed to access EBI')`